### ISSUE TITLE
textSize() leads to some symbols not being displayed correctly anymore

### ISSUE THREAD

**Issue by hkiel** [benfry/processing4#303]
## Description
`textSize()` leads to some symbols not being displayed correctly anymore with `text()`

## Expected Behavior
Same symbols should be shown with scaled font

## Current Behavior
glyph for unknown symbol is shown instead

## Steps to Reproduce
```
text("♦♥♠♣", 0, 20);
textSize(20);
text("♦♥♠♣", 0, 60);
```

![Bildschirmfoto 2021-11-04 um 13 19 13](https://user-images.githubusercontent.com/9904830/140312309-696b920e-d051-49e0-afe9-b580ec1063b6.png)

## Your Environment
* Processing version: 4.0b2
* Operating System and OS version: WIn10, macOS

**Comment by benfry** [comment]
Looks like the same issue as https://github.com/processing/processing/issues/4956. 

Since that repo is no longer being updated, here's the status of the issue: 

There are likely two things going on here (This is all inside `handleTextSize()` in `PGraphicsJava2D`)
1. The internal `deriveFont()` call seems to be not including that part of the character set properly. Might just need to include additional attributes/properties or something on those lines. 
2. We shouldn't be calling `deriveFont()` again on a font that's already the same size.

I've already fixed the second item in 3.x, but it's not clear why `deriveFont()` is killing off those characters. Though it may be that deriving the font causes it to go from the "logical" `SansSerif` font that Java uses, to a "physical" font that's a specific sans serif face, and doesn't include the necessary code points. More about fonts [here](https://docs.oracle.com/javase/tutorial/2d/text/fonts.html). 

The workaround is to specify an actual font, rather than relying on the default font.

**Comment by benfry** [comment]
Ah… I've at least found the problem. The bug (sort of) is that the first line isn't actually using the default font from Processing, it's using Java's built-in font. The second line correctly uses the Processing default, which doesn't support all of the characters.

So the workaround in your case would be to add this to the beginning of your sketch:

```processing
PFont font = createFont("SansSerif", 12)
textFont(font);
```

**Comment by benfry** [comment]
Resolved for 4.0 beta 3. For this example, it's still necessary to use `createFont()` with `SansSerif` or another  typeface that includes those characters, but at least the confusion has been removed. 

Also added a warning for the user to explain the situation in https://github.com/processing/processing4/commit/154d44eb59d91e031c7e86e669826797fd78d9a2
